index.html

```diff
@@ -0,0 +1,236 @@
+                                printer took a standard dummy text.</p>
+                            <ul class="tags">
+                                <li>html</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button class="action-btn">See Project</button>
+                    </div>
+                </div>
+    
+            </section>
+            <section class="about-me">
+                <div class="the-resume">
+                  <div class="myresume">
+                    <h3 class="title-h3">About me</h3>
+                    <p class="about-paragraph">
+                      Hello I’m a software developer! I can help you build a product ,
+                      feature or website Look through some of my work and experience! If
+                      you like what you see and have a project you need coded, don’t
+                      hestiate to contact me.
+                    </p>
+                    <button class="resume" type="button">Get My Resume</button>
+                  </div>
+                </div>
+                <di class="vector">
+                    <img src="./images/Vector 4.svg" alt="line" />
+                </di>
+                <div class="work-on">
+                  <div class="all-about">
+                    <h3 class="best">
+                      <img src="./images/Rectangle 55 (3).svg" alt="rectangl" /> Language
+                    </h3>
+                    <div class="lang">
+                        <ul class="list">
+                            <li><span>Javascript</span></li>
+                            <li><span>Ruby</span></li>
+                            <li><span>Html</span></li>
+                            <li><span>Css</span></li>
+                          </ul>
+                    </div>
+                    
+                  </div>
+                  <div class="all-about">
+                    <h3 class="best">
+                      <img src="./images/Rectangle 56.svg" alt="rectangle" /> Frameworks
+                    </h3>
+                    <div class="lang frame">
+                        <ul class="list">
+                            <li><span>Bootstrap</span></li>
+                            <li id="ruby"><span id="ru">Ruby on Rails</span></li>
+                            <li><span>Rspec</span></li>
+                            <li><span>CapyBara</span></li>
+                            <li><span>Selenium</span></li>
+                          </ul>                        
+                    </div>
+                    
+                  </div>
+                  <div class="all-about">
+                    <h3 class="best" id="skill">
+                      <img src="./images/Rectangle 57.svg" alt="circle" /> Skills
```

Comment:
- [ ] Please consider putting your technology categories into a list 📃. This way you make sure your website is always user-friendly and it's always more readable and makes more sense 🤝🏽. **_Here is an example_** 👇🏽

![image](https://user-images.githubusercontent.com/43172164/199302442-64a940be-716a-4285-8f49-f3db41ee14e5.png)